openfecli/commands/gather.py

```diff
@@ -96,8 +96,25 @@ def load_results(fpath:os.PathLike|str)->dict:
     import json
     from gufe.tokenization import JSON_HANDLER
 
-    return json.load(open(fpath, 'r'), cls=JSON_HANDLER.decoder)
+    try:
+        result = json.load(open(fpath, 'r'), cls=JSON_HANDLER.decoder)
+    except FileNotFoundError:
+        click.echo(f"Error: {fpath} does not exist. Skipping.")
+        return None
 
+    if "unit_results" not in result.keys():
+        click.echo(f"{fpath}: No 'unit_results' found, assuming to be a failed simulation.", err=True)
+
+    if all('exception' in u for u in result['unit_results'].values()):
+        click.echo(f"{fpath}: Exception found in 'unit_results', assuming to be a failed simulation.", err=True)
+
+    if result['estimate'] is None:
+        click.echo(f"No{fpath}: No 'estimate' found, assuming to be a failed simulation", err=True)
+
+    if result['uncertainty'] is None:
+        click.echo(f"{fpath}: No 'uncertainty' found, assuming to be a failed simulation", err=True)
```

Comment:
If an edge has failed this will print all of these messages for that edge? It would be best to have a single message per edge indicating if it has failed.

Reply:
I agree and that's the behavior now 👍